relay-server/src/services/store.rs

```diff
@@ -244,6 +231,23 @@ impl StoreService {
                         item,
                     )?;
                 }
+                ItemType::UserReportV2 => {
+                    // The preferred, newest form of feedback. Main source is user feedback widget
+                    let global_config = self.global_config.current();
+                    let feedback_ingest_topic_rollout_rate =
+                        global_config.options.feedback_ingest_topic_rollout_rate;
+                    let remote_addr = envelope.meta().client_addr().map(|addr| addr.to_string());
+
+                    self.produce_user_report_v2(
+                        event_id.ok_or(StoreError::NoEventId)?,
+                        scoping.organization_id,
+                        scoping.project_id,
+                        start_time,
+                        item,
+                        remote_addr,
+                        is_rolled_out(scoping.organization_id, feedback_ingest_topic_rollout_rate),
```

Comment:
It's better if you pass in the topic instead or even better, let the produce function figure out the topic.